From a.beregalov@gmail.com  Tue Jun  2 12:05:13 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Wed, 13 May 2009 17:00:47 +0400
Subject: Staging: comedi: comedi_test.c should include timer.h
To: gregkh@suse.de
Cc: Alexander Beregalov <a.beregalov@gmail.com>
Message-ID: <1242219648-16165-1-git-send-email-a.beregalov@gmail.com>


Fix this build error:
.../comedi_test.c:82: error: field timer has incomplete type
.../comedi_test.c: In function waveform_ai_interrupt:
.../comedi_test.c:188: error: implicit declaration of function mod_timer
.../comedi_test.c:188: error: jiffies undeclared (first use in this function)

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/comedi_test.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/comedi/drivers/comedi_test.c
+++ b/drivers/staging/comedi/drivers/comedi_test.c
@@ -55,6 +55,7 @@ zero volts).
 #include <asm/div64.h>
 
 #include "comedi_fc.h"
+#include <linux/timer.h>
 
 /* Board descriptions */
 struct waveform_board {
